concepts/observability-strategy/configurable-tracing/01-tracing-groundwork.md

```diff
@@ -0,0 +1,55 @@
+
+## Requirements
+
+### General
+- Users need a way to outstream trace data into multiple external systems. There will be no production-ready solution provided by Kyma, so user need to integrate with something.
+- Users need a way to add custom trace data from users' workload to that outstreams. Support for custom trace data is a major requirement for a distributed tracing solution.
+- Users need a way to influence the amount of traces being outstreamed. External systems will have a price dependent on load and traffic. Not relevant traces must be droppable with some strategies.
+
+### Basic backend configuration
+- Have a vendor-neutral layer of collectors that collects and ships trace data, but does not permanently store it (as a backend).
+- The collector must run stably at any time when using the typical settings. Bad configuration must be prevalidated and rejected. Fast feedback is welcome.
+- Outputs
+  - Support configuration of backends and outputs at runtime (no need to run a Kyma upgrade process) in a scenario-focused approach.
+  - Support multiple configurations at the same time (in individual Kubernetes resources) to support easy activation of dedicated scenarios.
+  - As a minimum, support one vendor-neutral output. It should be possible to chain your custom collector for specific conversions. For example, supporting the OLTP protocol will support most of the vendors already. Chaining a custom OpenTelemetry Collector can do custom conversion to a specific protocol.
+  - Support typical settings for the supported outputs in a convenient way.
+  - Non-typical settings should be possible as well, leaving the collector in an unsupported mode.
+  - Secrets must be kept secret.
+  - Secret bindings for the SAP BTP Operator must be supported, including secret rotations.
+  - Scenarios must be isolated and have their own buffer management. If one backend is in a bad shape and cannot process any data anymore, data should still continue to be pushed to other backends.
+  - Typical auth mechanisms for the integration must be supported, especially client-certificate-based solutions.
+- Inputs
+  - Inputs are predefined. In best case, users do not need to differentiate between inputs.
+- Filter
+  - Filtering of data (like dropping attributes or whole traces) must be possible.
+
+### Pre-integration
+- Kyma system components are pre-integrated, so the predefined input in the collector serves them by default (eventing and serverless mainly)
+- Istio and serverless trace data is pre-integrated and can be easily filtered and de-selected.
+
+### Ease of integration
+- It should be easy to integrate trace data of a workload - with a push approach to a well-known internal URL in a vendor-neutral protocol
+- Envoy trace data should be collected instantly (without user action).
+
+### Local backend
+- Kyma will provide a blueprint based on Helm for installing a Jaeger based Deployment.
```

Comment:
```suggestion
- Kyma will provide a blueprint based on Helm for installing a Jaeger-based Deployment.
```
"Deployment" is about the Kubernetes term, right? Otherwise, don't capitalize it.